spec/requests/api/v1/conversations/messages/index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe 'GET api/v1/conversations/{id}/messages', type: :request do
+  let(:user)            { create(:user) }
+  let!(:conversation)   { create(:conversation_with_messages, users: [user]) }
+
+  before do
+    get api_v1_conversation_messages_path(conversation),
+        headers: auth_headers, as: :json
+  end
+
+  it 'returns messages' do
+    conversation.messages.each do |message|
+      expect(json).to include_json(
+        messages: UnorderedArray(id: message.id,
```

Comment:
Just curious about what UnorderedArray is?

Reply:
It is a json-expectations matcher which tests whether a collection of elements are included into an array (in no particular order) included within a json.
Here are some examples: [https://relishapp.com/waterlink/rspec-json-expectations/docs/json-expectations/unordered-array-matching-support-for-include-json-matcher](https://relishapp.com/waterlink/rspec-json-expectations/docs/json-expectations/unordered-array-matching-support-for-include-json-matcher)